src/sentry/static/sentry/app/components/loadingError.tsx

```diff
@@ -23,36 +29,49 @@ class LoadingError extends React.Component<Props> {
   static propTypes = {
     onRetry: PropTypes.func,
     message: PropTypes.string,
+    isPanel: PropTypes.bool,
   };
 
   static defaultProps: DefaultProps = {
     message: t('There was an error loading data.'),
+    isPanel: false,
   };
 
   shouldComponentUpdate() {
     return false;
   }
 
   render() {
-    const {message, onRetry} = this.props;
+    const {message, isPanel, onRetry} = this.props;
     return (
-      <Alert type="error">
+      <StyledAlert isPanel={isPanel} type="error">
         <Content>
           <IconInfo size="lg" />
           <div data-test-id="loading-error-message">{message}</div>
           {onRetry && (
-            <Button onClick={onRetry} priority="default" size="small">
+            <Button onClick={onRetry} type="button" priority="default" size="small">
               {t('Retry')}
             </Button>
           )}
         </Content>
-      </Alert>
+      </StyledAlert>
     );
   }
 }
 
 export default LoadingError;
 
+const StyledAlert = styled(Alert, {
+  shouldForwardProp: prop => isPropValid(prop) && prop !== 'isPanel',
+})<{isPanel: boolean}>`
+  ${p =>
+    p.isPanel &&
+    css`
+      border-radius: 0;
+      border-width: 1px 0;
+    `}
+`;
```

Comment:
I think what we can do here is instead of relying on a prop, we can use `<Panel>` as a selector since we are exporting a styled component. So we select the `<Alert>` when it is a child of `<Panel>`:

```
const StyledAlert = styled(Alert)`
  ${Panel} & {
	  border-radius: 0;
      border-width: 1px 0;
  }
`;
```


We should also get rid of the bottom margin while we're at it (although I'm not sure if we have components inside of a `<Panel>` that renders `<LoadingError>` alongside other components)

Reply:
Docs: https://emotion.sh/docs/nested

Reply:
Thanks for the suggestion, this feels better than the prop. 

I opted to keep the margin as removing the margin started to look odd even on the issues stream, because then it needs a border radius on the bottom to match the panel.

Reply:
👍 We should look into a follow-up PR that styles the loading error when used in the issues stream. Would love to get rid of that bottom margin